Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchDeviceManager.cs

```diff
@@ -41,7 +41,8 @@ public class UnityTouchDeviceManager : BaseInputDeviceManager
         /// <inheritdoc />
         public override void Update()
         {
-            for (var i = 0; i < UInput.touches.Length; i++)
```

Comment:
Was this also causing GC alloc? Usually I would see this when creating enumerators (i.e. using foreach) so this would be somewhat surprising.

Reply:
Yeah it was creating an array each time touches was called.

Reply:
Danggggggggggggggggggggggggggggggggggggggg